data/zilliqa/assets/zil1nhuzjg452rkdzrfh7swllr4eq2g8t0lfgu2z38/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Planetarize",
+  "symbol": "PLANET",
+  "supply": "200000",
+  "email": "zilliqaplanetarize@gmail.com",
+  "links": {
+    "telegram": "https://t.me/ZilliqaPlanetarize",
+    "twitter": "https://twitter.com/PlanetarizeZ",
+    "github": "https://github.com/PlanetarizeZilliqa"
```

Comment:
since there is no repo related to the project, just remove this and revist later

Reply:
yes sir